### PR TITLE
content(cms): create Post "basics-of-tei/index"

### PR DESCRIPTION
**Pull request by stefanprobst**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/posts/basics-of-tei/index.mdx [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+[^1]: Body bezeichnet den Fließtext. Gemeinsam mit dem Header bildet er das TEI-Dokument.
+[^2]: `<div>`-Element steht für division, eine obere Strukturebene, die einen Abschnitt, der von anderen Textabschnitten abgesetzt ist, bezeichnet. Innerhalb eines `<div>`-Elements können mehrere Elemente sitzen, zum Beispiel eine Überschrift `<head>` und ein Absatz `<p>`.
+[^3]: Header bezeichnet die Metadaten über den Text, die nicht Teil des Fließtextes sind. Gemeinsam mit dem Body bildet er das TEI-Dokument.
+[^4]: wohlgeformte Elemente wurden geöffnet und wieder geschlossen, zum Beispiel `<head>` Trainspotting `</head>`. Wohlgeformte Elemente werden immer auf kleinster Ebene geschlossen, bevor sie in eine größere Struktur eingebettet sind.
+[^5]: valide Elemente sind nicht nur wohlgeformt, sondern auch konform zum festgelegten XML Schema.
+
+### Weiterführende Quellen:
+
+nicht notwendig [^5]: Die vollständige Annotation dieses Schrittes finden Sie in Datei `3_Struktur_II_FW_11_12.xml`
```
Comment (stefanprobst) [inline content/posts/basics-of-tei/index.mdx:423]: ?

File: content/posts/basics-of-tei/index.mdx [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+
+Die Auszeichnung verändert nicht die Anordnung der einzelnen Textteile, aber durch Einrückungen wird gezeigt, wie die Textteile zueinander strukturiert sind. `<div>` bildet die oberste Hierarchieebene, darin folgen dann `<head>`, `<p>`, etc als weitere Ebenen. Sie können gleichwertig sein, oder weiter ineinander verschachtelt. Bei Aufzählungen wird die oberste Hierarchieebene mit `<list>` gekennzeichnet, innerhalb der `<list>` findet man einen oder mehrere `<item>`.
+
+Beachten Sie, dass der Fließtext in einem Element von einem kleineren `<head>`-, `<p>`- bzw. `<item>` oder `<l>`-Element umgeben sein muss.
+
+Zusätzlich zur Identifizierung einzelner Abschnitte im Fließtext (Überschrift, Absatz, Strophe, Liste,...), zeichnet man im Body auch mit Attributen und Werten [^6] aus, worum es sich bei dem Element konkret handelt (z.B. Gedicht, Spielplan, Artikel,...).
+
+* `<div>` + type article / movieSchedule / poem / editorialStatement / reviews / review
+* `<div>` type review + n 1 / 2
+* `<head>` + type columnTitle [^5]
```
[the remark this answers is not in the document]
Comment (stefanprobst) [reply content/posts/basics-of-tei/index.mdx:118]: ok, actually footnote ref 6 (above) does not have a definition

File: content/posts/basics-of-tei/index.mdx [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+#### Übung 1a: Fließtextabschnitte auszeichnen
+
+In einer späteren Analyse sollen alle Überschriften getrennt von Artikeln aus mehreren Zeitschriftenausgaben gefunden werden können, um ein Verzeichnis aller Titel zu erstellen. Wir beginnen mit einem Beispieltext. Welche Fließtextabschnitte können Sie erkennen? Fügen Sie `<div>` (Abschnitt), `<head>` (Überschrift) und `<p>` (Absatz) an den richtigen Stellen ein. Vergessen Sie nicht, die Auszeichnungen zu schließen, sodass jedes Textelement umschlossen ist. Bitte beachten Sie, dass Überschrift und Absatz gemeinsam einen Abschnitt bilden sollen. Die umschließenden `<xml>`-Tags und der Fließtext sind bereits vorgegeben.
+
+![Die Wunder des Meeres, Überschrift](images/filmwelt_1923_31_11_titel.png "Die Wunder des Meeres, Überschrift")
+
+![Die Wunder des Meeres, Text](images/filmwelt_1923_31_11_text.png "Die Wunder des Meeres, Text")
+
+<Quiz>
+  <Quiz.Card validateButtonLabel="Überprüfen">
```
[the remark this answers is not in the document]
Comment (ekoenigshofer) [reply content/posts/basics-of-tei/index.mdx:298]: I'm happy to use Überprüfen for Validate

**Review by ekoenigshofer — APPROVED** [review]

**Comment by stefanprobst** [comment]
going to close this as it seems to have gone stale. feel free to reopen, or create a new one, if this is still being worked on.

**Pull request closed without merging**